core/adapter/btc-usdt.adapter.json

```diff
@@ -1,241 +1,303 @@
 {
   "id": "0x193a5f196fce5b0cc022f7c298b71d425886e062a3f3401db68f7df4ae6ed9c1",
-  "active": true,
```

Comment:
Question: since we removed `active` status, does this settigs moved somewhere else? , or we assume all of the adapters are active.

Reply:
Yes. It moved to aggregators where I think it has already been before anyway. Currently, there are outdated checks on structure of adapters and aggregators, but it will be soon moved under Orakl Network API.

https://github.com/Bisonai/orakl/blob/master/core/aggregator/localhost/btc-usdt.aggregator.json

The reason we had `active` in adapters is from time when we were planning to offer predefined data query jobs.